components/datetime/src/neo_skeleton.rs

```diff
@@ -78,6 +79,26 @@ impl NeoSkeletonLength {
     fn to_components_week_of_year(self) -> components::Week {
         components::Week::NumericWeekOfYear
     }
+
+    /// Returns the date style corresponding to this length.
+    pub fn to_date_style(self) -> options::length::Date {
+        match self {
+            Self::Long => options::length::Date::Long,
+            Self::Medium => options::length::Date::Medium,
+            Self::Short => options::length::Date::Short,
+        }
+    }
+
+    /// Returns the time style corresponding to this length.
+    pub fn to_time_style(self) -> options::length::Time {
```

Comment:
thought: all of these could be Into impls perhaps (non blocking, not necessary to do at all)